entries/d/deviantart.com.json

```diff
@@ -3,7 +3,8 @@
     "domain": "deviantart.com",
     "contact": {
       "twitter": "DeviantArt",
-      "facebook": "deviantart"
+      "facebook": "deviantart",
+      "form": "https://www.deviantartsupport.com/en/article/product-feedback-and-feature-requests"
```

Comment:
Should forms be behind login screens?

Reply:
I would argue that in order to contact them (regardless of the method you choose) you need some sort of account, whether that be an email, Twitter or Facebook account. Plus people requesting 2FA I would assume 9 times out of 10 already have an account with the site/service.

Reply:
Good argument